examples/SubstitutionCast.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { Sheet, TextField, Workbook, SubstitutionCast } from '@flatfile/configure'
+import { SheetTester } from '../src/utils/testing/SheetTester'
+
+
+
+
+const spanishNumeralSynonyms = [
+  ['1', 'one', 'un'],
+  ['2', 'two', 'dos'],
+]
```

Comment:
Suggestion: separate this into two files and flesh out more fully:
* `examples/fields/SpanishNumberField.ts` (using structure from #36). This file will have a fully functional Spanish Number Field using SubstitutionCast. Expand the spanishNumeralSynonyms to include all numerals 0-9 and use the correct Spanish words ("uno" as well as "un").
* `examples/fields/SpanishNumberFieldSpec.ts`: the tests you have in this file

That structure will make it more clear to the user how they would actually write / use / test a custom field like this in their own code.